src/test/raii_event_tests.cpp

```diff
@@ -4,8 +4,18 @@
 
 #include <event2/event.h>
 
+#include <boost/test/unit_test.hpp>
+
+BOOST_FIXTURE_TEST_SUITE(raii_event_tests, BasicTestingSetup)
+
+#ifndef EVENT_SET_MEM_FUNCTIONS_IMPLEMENTED
```

Comment:
Maybe consider making this an #else later on (or at least turn the existing #ifdef into an #else if we prefer the skip-test on top)